### ISSUE TITLE
Internal error while querying node's versions

### ISSUE THREAD

**Issue by DevNils** [gentics/mesh#1117]
## Gentics Mesh Version, operating system, or hardware.
- v1.7.0

## Operating System
- MacOSX
- Docker

## JVM
- Open JDK 1.8.0_100

## Problem
On some (not all) of my nodes I get an error when I try to query the node's versions:
```
08:54:44.810 [] ERROR [vert.x-eventloop-thread-8] - Error for request in path: /api/v2/entdeckertour/nodes/1bcedad277a34298884ab11fcf8678ba/versions?lang=en
08:54:44.819 [] ERROR [vert.x-eventloop-thread-8] - Error:
java.lang.NullPointerException: null
	at com.gentics.mesh.core.data.container.impl.NodeGraphFieldContainerImpl.transformToVersionInfo(NodeGraphFieldContainerImpl.java:797)
	at com.gentics.mesh.core.data.node.impl.NodeImpl.lambda$transformToVersionList$11(NodeImpl.java:1003)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.base/java.util.Iterator.forEachRemaining(Unknown Source)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
	at com.gentics.mesh.core.data.node.impl.NodeImpl.lambda$transformToVersionList$12(NodeImpl.java:1004)
	at java.base/java.lang.Iterable.forEach(Unknown Source)
	at com.gentics.mesh.core.data.node.impl.NodeImpl.transformToVersionList(NodeImpl.java:1001)
	at com.gentics.mesh.core.endpoint.node.NodeCrudHandler.lambda$handleListVersions$43(NodeCrudHandler.java:502)
	at com.gentics.mesh.graphdb.OrientDBDatabase.tx(OrientDBDatabase.java:416)
	at com.gentics.mesh.core.verticle.handler.HandlerUtilities.syncTx(HandlerUtilities.java:233)
	at com.gentics.mesh.core.endpoint.node.NodeCrudHandler.handleListVersions(NodeCrudHandler.java:500)
	at com.gentics.mesh.core.endpoint.node.NodeEndpoint.lambda$addVersioningHandlers$1(NodeEndpoint.java:143)
	at io.vertx.ext.web.impl.BlockingHandlerDecorator.lambda$handle$0(BlockingHandlerDecorator.java:48)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Unknown Source)
08:54:44.834 [] ERROR [vert.x-eventloop-thread-8] - 172.18.0.4 - GET /api/v2/entdeckertour/nodes/1bcedad277a34298884ab11fcf8678ba/versions?lang=en HTTP/1.1 500 44 - 47 ms
```

## Reproducer
`/api/v2/entdeckertour/nodes/1bcedad277a34298884ab11fcf8678ba/versions?lang=en`
I don't really know when this error occurred the first time and how to reproduce it. During development I fiddled around with users, deleted and added them. It might be possible that some users are not existing anymore, that where responsible for some versions.

Is it possible that the missing users are causing this error?

## Expected behaviour and actual behaviour
Should deliver the version history of the node.

**Comment by Jotschi** [comment]
NPE in `info.setCreator(getEditor().transformToReference());`

TODO:
* Check whether user could be deleted
* Add NPE check for getEditor()==null

**Comment by Jotschi** [comment]
Thanks for reporting the issue. I made a fix for the error which will be included in the next release of Gentics Mesh.

**Comment by DevNils** [comment]
Thank You! You are welcome.

**Comment by Jotschi** [comment]
Fixed with version 1.7.1
